shared/services/rocketpool/client.go

```diff
@@ -60,6 +61,23 @@ const (
 	DebugColor = color.FgYellow
 )
 
+// Get the external IP address. Try finding an IPv4 address first to:
+// * Improve peer discovery and node performance
+// * Avoid unnecessary container restarts caused by switching between IPv4 and IPv6
+func getExternalIP() (net.IP, error) {
```

Comment:
I agree that external IP should be "stable" and have a deterministic ordering of obtaining an IP.

however, in this day and age with so much NAT, preferring IPv4 instead of IPv6 doesn't feel like the best option.
For example, I have dual-ip stack and I'd much rather discover that besu has poor ipv6 support and report bugs to besu than using IPv4 ..

I dunno if I can prove factually than ipv4 first is better or the opposite though..

Reply:
I know this has bitten some community members several times in the past, so at least for now, preferring IPv4 first will help more people than hurt. The client teams are certainly aware of their IPv6 limitations, so I'm inclined to do this for now in the name of supporting them until they fix it.